tests/src/test/java/alluxio/shell/AbstractAlluxioShellTest.java

```diff
@@ -69,6 +69,12 @@ public final void after() throws Exception {
 
   @Before
   public final void before() throws Exception {
+    if (mLocalAlluxioClusterResource.isStarted()) {
```

Comment:
this is needed (although refactored) because it is possible that we run this `before()` method before we really start the cluster after this PR where we explicitly launch cluster in some test cases after changing some conf.

About "inline the resetShellTest",  `resetShellTest` is needed in other tests too.